quickstart.js

```diff
@@ -52,16 +52,18 @@ async function getServiceAccessToken() {
 }
 
 
-async function getFromApi(serviceAccessToken, resourceUrl, queryParams = {}, raiseError = true) {
+async function getFromApi(serviceAccessToken, resourceUrl, queryParams = null, raiseError = true) {
 
-  const url = `${baseUrl}${resourceUrl}`
+  const queryString = queryParams ? `?${new URLSearchParams(queryParams)}` : '';
+  const url = `${baseUrl}${resourceUrl}${queryString}`;
+  console.log(url);

```

Comment:
We should either remove this log or add some clarity text if you feel it's important for debugging. Like:  "Calling ${url}"